src/js/tracks/text-track-display.js

```diff
@@ -319,6 +320,17 @@ class TextTrackDisplay extends Component {
       }
       this.updateForTrack(descriptionsTrack);
     }
+
+    if (browser.IS_SMART_TV && !window.CSS.supports('inset', '10px')) {
+      const textTrack = window.document.querySelector('.vjs-text-track-display');
```

Comment:
Can you just use:
```suggestion
      const textTrack = this.el_;
```

Reply:
this was added in the last commit, can this be re-reviewed?